pkg/features/kube_features.go

```diff
@@ -609,7 +609,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	ServiceNodeExclusion:           {Default: false, PreRelease: featuregate.Alpha},
 	NodeDisruptionExclusion:        {Default: false, PreRelease: featuregate.Alpha},
 	CSIDriverRegistry:              {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.20
-	CSINodeInfo:                    {Default: true, PreRelease: featuregate.GA, LockToDefault: true}, // remove in 1.19
+	CSINodeInfo:                    {Default: true, PreRelease: featuregate.GA},
```

Comment:
I believe the comment is referring to removing the entire feature gate.

Reply:
The storage version was already switched over in https://github.com/kubernetes/kubernetes/pull/84477, although I see we haven't done it for CSIDriver yet